### PR TITLE
Delete additional resources during uninstall, remove duplicate call to obtain rancher access token

### PR DESCRIPTION
**Pull request by bhatpmk**
# Description

This change attempts to address the issues we have seen in obtaining the rancher access token in the uninstall CI pipeline.

- Clean up additional Rancher resources based on the documentation at https://rancher.com/docs/rancher/v2.0-v2.4/en/cluster-admin/cleaning-cluster-nodes during uninstall
- The function install_verrazzano in 3-install-verrazzano.sh doesn't need to get the rancher access token and admin token, so that is deleted. The jobs mentioned in the issue has failures to get the rancher access token from this function, but succeeded in 2-install-system-components. I don't know whether obtaining the access token after setting the server url has any known issue.
- Re-arrange the kubectl wait for the pods to reach READY state
- Delete the unused code to get the rancher admin and access token, from the function which uninstalls Verrazzano
- The Jenkins scripts used by Verrazzano uninstall and loop pipelines are updated to capture more debugging information, in case of failure during Verrazzano install / reinstall

Contributes to VZ-2860

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

**Review by michael-cico — DISMISSED** [review]

File: ci/looping-test/Jenkinsfile [hunk 1]
```diff
@@ -535,3 +536,27 @@ def listNamepacesAndPods(customMessage) {
         echo "-----------------------------------------------------"
     """
 }
+
+def dumpResourcesOnFailure(customMessage) {
```
Comment (michael-cico) [inline ci/looping-test/Jenkinsfile:540]: should this stuff be in the dump script?

File: ci/uninstall/Jenkinsfile [hunk 2]
```diff
@@ -590,3 +592,27 @@ def getEffectiveDumpOnSuccess() {
     }
     return effectiveValue
 }
+
+def dumpResourcesOnFailure(customMessage) {
```
Comment (michael-cico) [inline ci/uninstall/Jenkinsfile:596]: same as the above...or just put into a separate helper script so it's not duplicated in 2 places.

**Review by michael-cico — APPROVED** [review]

File: ci/looping-test/Jenkinsfile (hunk 1 above)
Comment (bhatpmk) [reply ci/looping-test/Jenkinsfile]: I have moved this to a new script for now.

File: ci/uninstall/Jenkinsfile (hunk 2 above)
Comment (bhatpmk) [reply ci/uninstall/Jenkinsfile]: As mentioned above, I have added a new script to dump additional information.

**Review by michael-cico — APPROVED** [review]

**Review by wmhopkins — APPROVED** [review]
lgtm

**Comment by bhatpmk** [comment]
> lgtm

Thank you, @wmhopkins

**Pull request merged**